app-admin/kubelet-wrapper/files/kubelet-wrapper

```diff
@@ -4,36 +4,74 @@
 # Make sure to set KUBELET_VERSION to an image tag published here:
 # https://quay.io/repository/coreos/hyperkube?tab=tags Alternatively,
 # override $KUBELET_ACI to a custom location.
+#
 
 set -e
 
-if [ -z "${KUBELET_VERSION}" ]; then
-    echo "ERROR: must set KUBELET_VERSION"
-    exit 1
+function require_ev_all() {
+	for rev in $@ ; do
+		if [[ -z ${!rev} ]]; then
+			echo ${rev} is not set
+			exit 1
+		fi
+	done
+}
+
+function require_ev_one() {
+	for rev in $@ ; do
+		if [[ ! -z ${!rev} ]]; then
+			return
+		fi
+	done
+	echo One of $@ must be set
+	exit 1
+}
+
+if [[ ! -z ${KUBELET_VERSION} ]]; then
+	echo KUBELET_VERSION environment variable is deprecated, please use KUBELET_IMG_TAG instead
+fi
+
+if [[ ! -z ${KUBELET_ACI} ]]; then
+	echo KUBELET_ACI environment variable is deprecated, please use the KUBELET_IMG instead
 fi
 
-KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+KUBELET_IMG_TAG="${KUBELET_IMG_TAG:-${KUBELET_VERSION}}"
+KUBELET_IMG="${KUBELET_ACI}"
```

Comment:
I think this is incorrect. KUBELET_ACI is equivalent to KUBELET_IMG_URL. See usage: 
https://github.com/coreos/coreos-kubernetes/blob/master/single-node/user-data#L11
https://github.com/coreos/coreos-kubernetes/blob/master/single-node/user-data#L88